Classes/Command/HelpCommandController.php

```diff
@@ -209,22 +205,22 @@ protected function buildCommandsIndex()
     {
         $availableCommands = $this->commandManager->getAvailableCommands();
         /** @var RunLevel $runLevel */
-        $runLevel = ConsoleBootstrap::getInstance()->getEarlyInstance(\Helhum\Typo3Console\Core\Booting\RunLevel::class);
+        $runLevel = ConsoleBootstrap::getInstance()->getEarlyInstance('Helhum\Typo3Console\Core\Booting\RunLevel');
+
         foreach ($availableCommands as $command) {
             if ($command->isInternal()) {
                 continue;
             }
-            $commandIdentifier = $command->getCommandIdentifier();
-            $extensionKey = strstr($commandIdentifier, ':', true);
-            $commandControllerClassName = $command->getControllerClassName();
-            $commandName = $command->getControllerCommandName();
 
             $shortCommandIdentifier = $this->commandManager->getShortestIdentifierForCommand($command);
+
             if ($runLevel->getMaximumAvailableRunLevel() === RunLevel::LEVEL_COMPILE && !$runLevel->isCommandAvailable($shortCommandIdentifier)) {
                 continue;
             }
 
-            $this->commandsByExtensionsAndControllers[$extensionKey][$commandControllerClassName][$commandName] = $command;
+            $this->commands[$shortCommandIdentifier] = $command;
```

Comment:
Sure thing, let me know. The idea was that all commands could be sorted alphabetically if there is no extension name which makes it easier to find them.